services/storage/tests/unit/test_dsm_dsmcleaner.py

```diff
@@ -94,6 +94,7 @@ async def test_clean_expired_uploads_aborts_dangling_multipart_uploads(
         (LinkType.PRESIGNED, False),
     ],
 )
+@pytest.mark.parametrize("add_checksum", [True, False])
```

Comment:
Just declare `_FAKER = Faker()` as a global, we use this convention elsewhere.
This is a bit more explicit and compact. I would use this pattern elsewhere as well. See the 3 suggestions below that go with it.

```suggestion
@pytest.mark.parametrize("checksum", [None, _FAKER.sha256()])
```

Reply:
Good point. I fixed this.